util.go

```diff
@@ -13,6 +13,7 @@ func UnitDataAsBuildPackage(u *unit.SourceUnit) (*build.Package, error) {
 	if err := json.Unmarshal(u.Data, &pkg); err != nil {
 		return nil, err
 	}
+	pkg.Dir = filepath.Join(cwd, pkg.Dir)
```

Comment:
Usually, `build.Package#Dir` is always absolute. However, when we serialize the struct we make it relative so the output is deterministic. This is the reverse operation, making it absolute again, because that's what someone familiar with `build.Package` expects (I tripped over it).